src/commonTest/resources/dev/rhovas/interpreter/programs/rosettacode/Fibonacci.rho

```diff
@@ -0,0 +1,26 @@
+func pfib(n: Integer): Integer {
+    require n >= 0;
+
+    var lastFib = -1;
+    var fib = 1;
+    range(0, n, :incl).for {
+        var sum = fib + lastFib;
+        lastFib = fib;
+        fib = sum;
+    };
+    return fib;
+}
+
+func fib(n: Integer): Integer {
```

Comment:
Lets use `fibonacci` here; I think it's better to avoid abbreviated names in this case. Also, see the above note on moving the helper function into this one.